homeassistant/util/unit_conversion.py

```diff
@@ -742,6 +742,8 @@ class UnitlessRatioConverter(BaseUnitConverter):
     }
     VALID_UNITS = {
         None,
+        CONCENTRATION_PARTS_PER_BILLION,
+        CONCENTRATION_PARTS_PER_MILLION,
```

Comment:
I've considered moving those 2 units out of `UnitlessRatioConverter` into a new `PartsConcentrationConverter`, since they are not unitless. The downside is that it would be a breaking change for `recorder/statistic_during_period` and `recorder/statistics_during_period` commands: https://github.com/home-assistant/core/blob/fix_ppm_conversion/homeassistant/components/recorder/websocket_api.py#L63-L63